WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveyViewControllerTests.swift

```diff
@@ -118,19 +141,36 @@ private extension SurveyViewControllerTests {
 private extension SurveyViewControllerTests {
     final class FormSubmittedNavigationAction: WKNavigationAction {
 
-        private let httpMethod: String
-        init(httpMethod: String) {
-            self.httpMethod = httpMethod
+        static var emptyAction: FormSubmittedNavigationAction {
+            return FormSubmittedNavigationAction(messageTag: nil)
+        }
+
+        static var nonCompletedAction: FormSubmittedNavigationAction {
+            return FormSubmittedNavigationAction(messageTag: "other")
+        }
+
+        static var completedAction: FormSubmittedNavigationAction {
+            return FormSubmittedNavigationAction(messageTag: "done")
+        }
+
+        private let messageTag: String?
```

Comment:
Not important. "Tag" may be a bit unclear since `msg` is really a [query parameter](https://branch.io/glossary/query-parameters/). Maybe we can name it as such?

Reply:
Yeah, I used `Tag` because that is how CrowdSignal docs refer to it. So I had that terminology in mind. But I'll rename it.